### PR TITLE
Hcmpre 469 pagination all

### PR DESCRIPTION
**Pull request by kanishq-egov**

### CODE REVIEW

File: health-services/facility/src/main/java/org/egov/facility/repository/FacilityRepository.java [hunk 1]
```diff
@@ -83,12 +85,18 @@ public List<Facility> find(FacilitySearch searchObject, Integer limit, Integer o
         if (lastChangedSince != null) {
             query = query + "and lastModifiedTime>=:lastModifiedTime ";
         }
-        query = query + "ORDER BY f.id ASC LIMIT :limit OFFSET :offset";
+        query = query + "ORDER BY f.id ASC";
```
Comment (kavi-egov) [inline health-services/facility/src/main/java/org/egov/facility/repository/FacilityRepository.java:88]: order by shouldn't be there in count query

File: health-services/libraries/health-services-common/pom.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <artifactId>health-services-common</artifactId>
     <packaging>jar</packaging>
     <name>health-services-common</name>
-    <version>1.0.19-SNAPSHOT</version>
+    <version>1.0.20-dev-SNAPSHOT</version>
```
Comment (kavi-egov) [inline health-services/libraries/health-services-common/pom.xml:11]: update the library and merge to master

File: health-services/referralmanagement/pom.xml [hunk 3]
```diff
@@ -46,12 +46,12 @@
 		<dependency>
 			<groupId>org.egov.common</groupId>
 			<artifactId>health-services-common</artifactId>
-			<version>1.0.18-SNAPSHOT</version>
+			<version>1.0.20-dev-SNAPSHOT</version>
```
Comment (kavi-egov) [inline health-services/referralmanagement/pom.xml:49]: remove dev, update library and merge to master .

**Review by kavi-egov — CHANGES_REQUESTED** [review]

File: health-services/facility/src/main/java/org/egov/facility/web/controllers/FacilityApiController.java [hunk 4]
```diff
@@ -111,15 +112,17 @@ public ResponseEntity<FacilityBulkResponse> facilityV1SearchPost(
...
                 urlParams.getLimit(),
                 urlParams.getOffset(),
                 urlParams.getTenantId(),
                 urlParams.getLastChangedSince(),
                 urlParams.getIncludeDeleted());
         FacilityBulkResponse response = FacilityBulkResponse.builder().responseInfo(ResponseInfoFactory
-                .createResponseInfo(request.getRequestInfo(), true)).facilities(facilities).build();
+                .createResponseInfo(request.getRequestInfo(), true))
+                .facilities(searchResponse.getResponse())
+                .totalCount(searchResponse.getTotalCount()).build();
```
Comment (shubhang-eGov) [inline health-services/facility/src/main/java/org/egov/facility/web/controllers/FacilityApiController.java:125]: Total count will be set 0 for ids search, it should be facility.size()

Comment (Shashwat12-egov) [reply health-services/facility/src/main/java/org/egov/facility/web/controllers/FacilityApiController.java:125]: Done

File: health-services/facility/src/main/java/org/egov/facility/repository/FacilityRepository.java (hunk 1 above)
Comment (Shashwat12-egov) [reply health-services/facility/src/main/java/org/egov/facility/repository/FacilityRepository.java:88]: Removed

File: health-services/libraries/health-services-common/pom.xml (hunk 2 above)
Comment (Shashwat12-egov) [reply health-services/libraries/health-services-common/pom.xml:11]: Done

File: health-services/referralmanagement/pom.xml (hunk 3 above)
Comment (Shashwat12-egov) [reply health-services/referralmanagement/pom.xml:49]: done

**Review by kavi-egov — APPROVED** [review]

**Pull request merged**